locales/en/messages.json

```
@@ -2875,7 +2875,10 @@
         "message": "CLI reboot detected"
     },
     "cliSaveToFileBtn": {
-        "message": "Save to File"
+        "message": "Export to file"
```

Please, add a "description" field to all the messages. This description is used by the translators to get more information about the string. Something like:
 ```suggestion
         "message": "Export to file",
         "description": "Option in the CLI tab to export the contents to a file"
```

---

done!